docs/sources/installation/mac.md

```diff
@@ -1,116 +1,315 @@
+Docker natively in OS X. Instead, you must install the Boot2Docker application.
+The application includes a VirtualBox Virtual Machine (VM), Docker itself, and the
+Boot2Docker management tool.
 
-Because the Docker Engine uses Linux-specific kernel features, you'll need to use a
-lightweight virtual machine (VM) to run it on OS X. You use the OS X Docker client to
-control the virtualized Docker Engine to build, run, and manage Docker containers.
+The Boot2Docker management tool is a lightweight Linux virtual machine made
+specifically to run the Docker daemon on Mac OS X. The VirtualBox VM runs
+completely from RAM, is a small ~24MB download, and boots in approximately 5s.
 
-To make this process easier, we've built a helper application called
-[Boot2Docker](https://github.com/boot2docker/boot2docker) that installs a
-virtual machine (using VirtualBox) that's all set up to run the Docker daemon.
+**Requirements**
 
-## Demonstration
+Your Mac must be running OS X 10.6 "Snow Leopard" or newer to run Boot2Docker.
 
-<iframe width="640" height="360" src="//www.youtube.com/embed/wQsrKX4588U?rel=0" frameborder="0" allowfullscreen></iframe>
 
-## Installation
+## Learn the key concepts before installing
+ 
+In a Docker installation on Linux, your machine is both the localhost and the
+Docker host. In networking, localhost means your computer. The Docker host is
+the machine on which the containers run. 
 
-1. Download the latest release of the [Docker for OS X Installer](
-   https://github.com/boot2docker/osx-installer/releases/latest) (Look for the
-   green Boot2Docker-x.x.x.pkg button near the bottom of the page.)
+On a typical Linux installation, the Docker client, the Docker daemon, and any
+containers run directly on your localhost. This means you can address ports on a
+Docker container using standard localhost addressing such as `localhost:8000` or
+`0.0.0.0:8376`.
 
-2. Run the installer by double-clicking the downloaded package, which will install a
-VirtualBox VM, Docker itself, and the Boot2Docker management tool.
-   ![](/installation/images/osx-installer.png)
+![Linux Architecture Diagram](/installation/images/linux_docker_host.png)
 
-3. Locate the `Boot2Docker` app in your `Applications` folder and run it.
-   Or, you can initialize Boot2Docker from the command line by running:
+In an OS X installation, the `docker` daemon is running inside a Linux virtual
+machine provided by Boot2Docker.
 
-	     $ boot2docker init
-	     $ boot2docker start
-	     $ $(boot2docker shellinit)
+![OSX Architecture Diagram](/installation/images/mac_docker_host.png)
 
-A terminal window will open and you'll see the virtual machine starting up. 
-Once you have an initialized virtual machine, you can control it with `boot2docker stop`
-and `boot2docker start`.
+In OS X, the Docker host address is the address of the Linux VM.
+When you start the `boot2docker` process, the VM is assigned an IP address. Under
```

Comment:
ug. more conflation. - `when you start the Boot2Docker VM, is is assigned an IP address` - you don't need to use the `boot2docker` management tool to start the VM, and you can start the `boot2docker` management tool with other parameters that don't involve the VM at all